lib/battlestation/cli.rb

```diff
@@ -166,43 +168,75 @@ def install_packages(current_dirname)
 
     def verify_rbenv
       system 'bash', '-c', %{
-curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-doctor | bash
+curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-doctor | bash
+      }
+    end
+
+    # Runs the fzf install script in order to install key bindings and shell completion.
+    def configure_fzf
+      system 'bash', '-c', %{
+#{homebrew_prefix}/opt/fzf/install --key-bindings --completion --no-update-rc >/dev/null 2>&1
+if [[ "$?" -ne 0 ]]; then
+  echo 'Failed to install fzf'
+  exit 1
+fi
       }
     end
 
     # Installs Ruby if necessary.
     def install_ruby(version)
-      rbenv_versions_string = `rbenv versions`
+      rbenv_versions_string = `#{homebrew_prefix}/bin/rbenv versions`
 
       unless rbenv_versions_string.include? version
         system 'bash', '-c', %{
-rbenv init
-eval "$(rbenv init -)"
+eval "$(#{homebrew_prefix}/bin/rbenv init -)"
 
-rbenv install #{version}
+# Remove CFLAGS when openssl is fixed. See https://github.com/openssl/openssl/issues/18733#issuecomment-1181810055
+CFLAGS="-Wno-error=implicit-function-declaration" #{homebrew_prefix}/bin/rbenv install #{version}
         }
       end
     end
 
     def set_ruby_version(version)
       system 'bash', '-c', %{
-rbenv init
-eval "$(rbenv init -)"
+eval "$(#{homebrew_prefix}/bin/rbenv init -)"
 
-rbenv global #{version}
-rbenv shell #{version}
+#{homebrew_prefix}/bin/rbenv global #{version}
       }
     end
 
     def install_gems(current_dirname)
       system 'bash', '-c', %{
-rbenv init
-eval "$(rbenv init -)"
+eval "$(#{homebrew_prefix}/bin/rbenv init -)"
 
 gem install bundler
 cd "#{current_dirname}/../../"
 bundle install
       }
     end
+
+    # As part of this setup we install Homebrew and set up our dotfiles. Accordingly, a naked `brew`
```

Comment:
Currently we set up dotfiles first before we install packages though I recall seeing issues related to not being able to find `brew` when running setup on a new Apple Silicon Mac. It probably makes sense to install dotfiles last.